dashboard/lib/metrics/events.rb

```diff
@@ -33,8 +33,9 @@ def log_event(user: nil, event_name:, event_value: nil, metadata: {}, get_enable
 
         if CDO.rack_env?(:development)
           log_event_to_stdout(user: user, event_name: event_name, event_value: event_value, metadata: metadata, enabled_experiments: enabled_experiments)
+          log_statsig_event_with_cdo_user(user: user, event_name: event_name, event_value: event_value, metadata: metadata, enabled_experiments: enabled_experiments)
```

Comment:
This line should be removed right? Left over from local dev testing?

Reply:
Thanks, this might be a good data point for the upcoming work to add some sort of flag to enable development testing of statsig to avoid committing these!